Makefile

```diff
@@ -1,16 +1,36 @@
+DIST_NAME := popuko
+CONFIGURE_FILE := ./config.go
+
+all: help
+
 help:
 	@echo "Specify the task"
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 	@exit 1
 
-clean: ## remove virtualenv
-	rm -rf venv
+clean: ## Remove the exec binary.
+	rm -rf ./$(DIST_NAME)
+
+bootstrap:
```

Comment:
I'll plan to hear to @suzuken what most better approach we should do at this time. So this is temporary approach.